### ISSUE TITLE
Tests\Feature\Api\PostEntriesTest::testPostEntries ["filtering 'min_value'"] failing

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/money-tracker#43]
This [test](https://github.com/jdenoc/money-tracker/blob/455613cca644e93e5b95346f0e233dfcc09c30d0/tests/Feature/Api/PostEntriesTest.php#L100-L125) has failed with this error:
```
1) Tests\Feature\Api\PostEntriesTest::testPostEntries with data set "filtering 'min_value'" (array(44.539999999999999))
Failed asserting that a NULL is not empty.
/home/travis/build/jdenoc/money-tracker/tests/Feature/Api/ListEntriesBase.php:168
/home/travis/build/jdenoc/money-tracker/tests/Feature/Api/PostEntriesTest.php:124
````

This is the failed [build](https://travis-ci.org/jdenoc/money-tracker/builds/302005095).

**Comment by jdenoc** [comment]
Created #50 to try and identify the cause/problem.

**Comment by jdenoc** [comment]
Fixed by #69
